src/projection.ts

```diff
@@ -163,6 +163,12 @@ let setProperties = (
     return;
   }
   let eventHandlerInterceptor = projectionOptions.eventHandlerInterceptor;
+  let propertiesInterceptor = projectionOptions.propertiesInterceptor;
+
+  if (propertiesInterceptor) {
```

Comment:
updateProperties must also be kept in sync...
It is probably best performance-wise not to intercept when properties are the same...